licenses.d2iq.yaml

```diff
@@ -270,6 +270,151 @@ resources:
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/broker/filter@sha256:402fa0b876272bbcd03517d8a11088b543a0eb02c2a74ec65be16979f53c8dc2
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/broker/ingress@sha256:5f67e6ec7bd88293de294ba414ad74aedd17f3510bf33ab4d9ae9531379f2b29
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/controller@sha256:9828bd0a928620c2bb8ba3edb4bae8b97dad7b953cbda6caa4ae1d96ee3ff1ac
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/in_memory/channel_controller@sha256:1b42e21f9fe9f7e3e88f14f9c4f237f6273aa5378a96dd30a2aa1793a2c56f3c
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/in_memory/channel_dispatcher@sha256:bbf7c8bf1e550d5a8ba3cf145f4b0d27bdde4bf813449f825c7731b694fdf75b
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/jobsink@sha256:8cdeb8008975c7c8d67b4a3384b09ece25e85d8ad6acc6357ef31b1e2d44676b
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/mtchannel_broker@sha256:9176e09fc33d38a0ee7cc894b0267e34c9c87d2e0ea0c46ed64a2189f3ae012f
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/mtping@sha256:5611eb000a130082e6117b995919a0cd671dc963f9084b4726fa5dfeac916ce8
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/eventing/cmd/webhook@sha256:129fa9f552851b69d836df0a77236c78d2a0efe1bf2331226c3f31aef4e37d81
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
+        url: https://github.com/knative/eventing
+  - container_image: gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate@sha256:4446c18c763149a3ac93ab49868a8085c5163558b70542dfd6f2337ec3b4c985
+    sources:
+      - license_path: LICENSE
+        ref: knative-v1.18.1
```

Comment:
looks like there are no tags/releases on https://github.com/knative/pkg - they do have release branches like `release-1.18` https://github.com/knative/pkg/tree/release-1.18 if that can be used?

Reply:
Ok, should be good now.